pkgs/website/CLAUDE.md

```
@@ -230,49 +232,61 @@ For any page discussing features that aren't production ready:
    ```
 
 4. **Command Blocks**: Show commands without frames for easy copying
-   ```markdown
+
+   ````markdown
    ```bash frame="none"
    npx pgflow@latest install
    ```
+   ````
+
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)